auth_session_timeout/models/ir_config_parameter.py

```diff
@@ -14,23 +14,34 @@ class IrConfigParameter(models.Model):
     _inherit = 'ir.config_parameter'
 
     @tools.ormcache(skiparg=0)
-    def get_session_parameters(self, db):
+    def _auth_timeout_get_parameter_delay(self, db):
```

Comment:
This is changing the public external facing interface of a module that is already in production, and therefore is against a stable release policy.

Is the rename of this method actually necessary? If so, we'll need to provide a `get_session_parameters` method to use as an adaptation layer for old code.